packages/myst-cli/src/build/html/index.ts

```diff
@@ -89,7 +89,8 @@ function rewriteAssetsFolder(directory: string, baseurl?: string): void {
     const data = fs.readFileSync(file).toString();
     const modified = data
       .replace(new RegExp(`\\/${ASSETS_FOLDER}\\/`, 'g'), `${baseurl || ''}/build/`)
-      .replace('href="/favicon.ico"', `href="${baseurl || ''}/favicon.ico"`);
+      .replace('href="/favicon.ico"', `href="${baseurl || ''}/favicon.ico"`)
+      .replace('href="/myst-theme.css"', `href="${baseurl || ''}/myst-theme.css"`);
```

Comment:
Why did this work before?

Reply:
It didn't but there aren't many options, so didn't notice.